MC/run/ANCHOR/anchorMC.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+
+# add distortion maps
+# https://alice.its.cern.ch/jira/browse/O2-3346?focusedCommentId=300982&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-300982
+#
+# export O2DPG_ENABLE_TPC_DISTORTIONS=ON
+# SCFile=$PWD/distortions_5kG_lowIR.root # file needs to be downloaded
+# export O2DPG_TPC_DIGIT_EXTRA=" --distortionType 2 --readSpaceCharge ${SCFile} "
+
+#
+# procedure setting up and executing an anchored MC 
+#
+
+# make sure O2DPG + O2 is loaded
+[ ! "${O2DPG_ROOT}" ] && echo "Error: This needs O2DPG loaded" && exit 1
+[ ! "${O2_ROOT}" ] && echo "Error: This needs O2 loaded" && exit 1
+
+# ------ CREATE AN MC CONFIG STARTING FROM RECO SCRIPT --------
+# - this part should not be done on the GRID, where we should rather
+#   point to an existing config (O2DPG repo or local disc or whatever)
+export ALIEN_JDL_LPMANCHORYEAR=${ALIEN_JDL_LPMANCHORYEAR:-2022}
+RUNNUMBER=${ALIEN_JDL_LPMRUNNUMBER:-517616}
+#INTERACTIONRATE=${INTERACTIONRATE:-2000}
+
+# get the async script (we need to modify it)
+# the script location can be configured with a JDL option
+
+ALIEN_JDL_LPMPRODUCTIONTAG_KEEP=$ALIEN_JDL_LPMPRODUCTIONTAG
+echo "Substituting ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMPRODUCTIONTAG with ALIEN_JDL_LPMANCHORPRODUCTION=$ALIEN_JDL_LPMANCHORPRODUCTION for simulating reco pass..."
+ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMANCHORPRODUCTION
+
+# ZDC causes issues for sim
+#export ALIEN_JDL_WORKFLOWDETECTORS=ITS,TPC,TOF,FV0,FT0,FDD,MID,MFT,MCH,TRD,EMC,PHS,CPV,HMP,ZDC,CTP
+export ALIEN_JDL_WORKFLOWDETECTORS=ITS,TPC,TOF,FV0,FT0,FDD,MID,MFT,MCH,TRD,EMC,PHS,CPV,HMP,CTP
```

Comment:
shouldn't this list be dynamic and come from the data async scripts?
O2DPG already determines the list of detectors (somewhat) dynamically